igmpproxy.conf

```diff
@@ -33,9 +33,11 @@ phyint eth0 upstream  ratelimit 0  threshold 1
 
 
 ##------------------------------------------------------
-## Configuration for eth1 (Downstream Interface)
+## Configuration for eth1 (Downstream Interface),
+## blocking SSDP / UPnP traffic
 ##------------------------------------------------------
 phyint eth1 downstream  ratelimit 0  threshold 1
+        blacklist 239.255.255.250/32
```

Comment:
I do not like an idea to change default config file. I think that by default everything should be allowed and it is up to the admin to configure rounting as it is required per needs.

Reply:
I will revert this bit after discussion on the bahaviour of whitelist.

Reply:
Changes to the example config file have been reverted, and pushed.